source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,101 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a filter document. Calling the ``FindOne()``
+method retrieves a document from your MongoDB instance that matches your
+filter document. If you pass the ``FindOne()`` method an empty filter document,
+MongoDB considers your entire collection when retrieving your document. For more
+information on documents in the MongoDB Go driver, see our
+:doc:`guide on data formats </fundamentals/data-formats>`.
+
+You must pass the ``FindOne()`` method a ``context.Context`` interface type
+value. For more information on the ``context.Context`` interface and the
+``context`` package, see our :doc:`guide on context </fundamentals/context>`.
```

Comment:
I am not entirely sure, but "contexts" sounds more natural, since it's referring to the guide, and not quoting the title of the page.

```suggestion
``context`` package, see our :doc:`guide on contexts </fundamentals/context>`.
```

Reply:
Thanks for the note. I can't say I have a sense of which sounds more natural, but I trust your judgment. I haven't heard very many discussions of the `context` package so I can't say I have a great sense of how it is generally discussed in conversation. I will keep this edit in mind, and hopefully I will build up some intuition around when "context" vs "contexts" is more appropriate.